app/peerinfo/peerinfo.go

```diff
@@ -231,7 +231,7 @@ func (p *PeerInfo) sendOnce(ctx context.Context, now time.Time) {
 				peerCompatibleGauge.WithLabelValues(name).Set(0) // Set to false
 
 				// Log as error since user action required
-				log.Error(ctx, "Invalid peer version", err,
+				log.Error(ctx, "Peer is running an incompatible Charon version. Please coordinate with the operator to upgrade or downgrade to a compatible version", err,
```

Comment:
Not sure if we should use "operator" in Charon itself. I know we call entities that run Charon node operators, but I see that more as a business term, rather than code. IDK... Probably we can stick with peer?

```suggestion
				log.Error(ctx, "Peer is running an incompatible Charon version. Please coordinate with the peer to upgrade or downgrade to a compatible version", err,
```